### ISSUE TITLE
Ability to paste in server name or token.

### ISSUE THREAD

**Issue by abalter** [rschroll/juno#1]
I run notebooks on remote servers and use token security. Tokens are pretty hard to type in by hand. I'd like to be able to paste them in.

I suppose it could be a Mac glitch. I'm on a macbook pro running El Capitan.

**Comment by rschroll** [comment]
To see if I understand the problem: you're trying to paste a URL into the "Connect to server" entry on the "New Window" page?  And for some reason, nothing is being inserted when you paste?

On Ubuntu Linux, I'm able to paste both from the primary selection and the clipboard, so this looks to me an OS-dependent thing.  Perhaps this is relevant: https://pracucci.com/atom-electron-enable-copy-and-paste.html

**Comment by abalter** [comment]
Yes, that is exactly the problem! You were using the `CmdOrCtrl` keybindings. The problem is that electron doesn't have an "Edit" menu. I added

```
    {
      label: "Edit",
      submenu:
      [
        { label: "Cut", accelerator: "CmdOrCtrl+X", selector: "cut:" },
        { label: "Copy", accelerator: "CmdOrCtrl+C", selector: "copy:" },
        { label: "Paste", accelerator: "CmdOrCtrl+V", selector: "paste:" },
        { label: "Select All", accelerator: "CmdOrCtrl+A", selector: "selectAll:" }
      ]
    }
```

And that got paste to work!

**Comment by rschroll** [comment]
Can you checkout the clipboard branch and see if that solves things for you?
